### PR TITLE
add spanish

### PR DESCRIPTION
**Pull request by jblancogl**
Hi Chris, i´ve loved your project and i've translated it to spanish. Maybe you will be interested by the translation?
Thank you for the project,
Best

### CODE REVIEW

File: src/texts.js [hunk 1]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+				Pero las distracciones verdaderamente dañinas son las internas.
+				Puedes silenciar las notificaciones de Twitter y cerrar tu sesión de Slack, pero ¿cómo evitas que tu mente se distraiga?
+			</p>,
+
+			<p>
+				He perdido horas atrapado en webs que me parecían interesantes.
+				Lo más peligroso es cuando llega el dichoso instante en el que se me antoja preguntarme:
+				"¿cúal será la segunda lengua más hablada?"
+				Esos 500 milisegundos pueden cambiar tu día, porque al final nunca es una sola búsqueda en Google o un solo artículo en la Wikipedia.
+				Desconectándote de Internet cortocircuiteas estos caprichos, lo que te permite avanzar sin obstáculos.
```
Comment (sorodrigo) [inline src/texts.js]: Desconectarse de internet cortocircuita estos caprichos, permitiéndote avanzar sin obstáculos.

> Tone edit, more appropriate translations, cortocircuiteas is incorrect.

File: src/texts.js [hunk 2]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+			</p>,
+
+			<p>
+				(Es el español, por cierto.)
+			</p>,
+
+			<p>
+				Esta página es un experimento en esa línea: ¿Qué pasaría si para acceder a algunos contenidos fuese obligatorio desconectarse	?
```
Comment (sorodrigo) [inline src/texts.js]: [..] ¿Qué pasaría si para acceder a cierto contenido fuese obligatorio desconectarse?

> Tone edit

File: src/texts.js [hunk 3]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+				¿Qué pasaría si los lectores tuvieran acceso a esa gloriosa capacidad de atención que te permite devorar una novela durante horas y lo hace tan placentero?
```
Comment (sorodrigo) [inline src/texts.js]: [..] capacidad de atención que hace devorarse una novela durante horas tan placentero?

> More appropriate translations.

File: src/texts.js [hunk 4]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+				¿O si pudiéramos juntar a los creadores con el poder de los dispositivos modernos?
```
Comment (sorodrigo) [inline src/texts.js]: ¿Qué pasaría si los creadores pudieran juntar eso con el poder de los dispositivos modernos?

> Incorrect translation

File: src/texts.js [hunk 5]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+				Nuestros teléfonos y ordenadores portátiles son plataformas increíbles para crear contenidos siempre y cuando seamos capaces de concentrarnos.
```
Comment (sorodrigo) [inline src/texts.js]: Nuestros teléfonos y ordenadores portátiles son plataformas increíbles para crear contenido creativo—si tan solo pudiéramos explotar nuestra capacidad de atención.

> Incorrect translation

File: src/texts.js [hunk 6]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+			</p>,
+
+			<p>
+				El contenido disponible sólo en modo sin conexión también puede llevar a los creadores a pensar de manera diferente.
```
Comment (sorodrigo) [inline src/texts.js]: El contenido offline también obliga a los creadores a pensar de manera diferente.

> Incorrect translation

File: src/texts.js [hunk 7]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+				Mira esta página: no tiene ni un sólo enlace que distraiga a los lectores.
+				¿Cuántos buenos artículos has dejado a la mitad por haber seguido un enlace llamativo?
+				Cuando estás sin conexión, el único sitio donde puedes estar es aquí.
```
Comment (sorodrigo) [inline src/texts.js]: [..] donde puedes estar es aquí mismo.

> Tone edit (however both versions are good to me)

File: src/texts.js [hunk 8]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+			</p>,
+
+			<p>
+				Ya oigo las quejas: "Pero tengo que estar conectado para trabajar." Me da igual. Encuentra tiempo.
```
Comment (sorodrigo) [inline src/texts.js]: Ya puedo oír las quejas: "Pero tengo que estar conectado para trabajar." Me da igual. Haz tiempo.

> Incorrect translation.

File: src/texts.js [hunk 9]
```diff
@@ -189,6 +189,82 @@ export const languages = {
...
+				Ahora vuelve a tu actividad normal en Internet. Lo único, no te olvides hacerte de vez en cuando un regalo a tí mismo: desconectar.
+			</p>,
+
+			<p>
+				<span role="img" aria-label="love">❤️</span> Chris Bolin
+				<br/>
+				<i>(@bolinchris cuando te conectes)</i>
+			</p>,
+			<p>
+				Traducido por Javier Blanco García-Lomas.
```
Comment (sorodrigo) [inline src/texts.js]: Traducido por Javier Blanco García-Lomas y Rodrigo Solís.

> 😄  ✌️

**Review by sorodrigo — CHANGES_REQUESTED** [review]
I also thought of contributing with the Spanish translations. Great job!! I left some edits to stick closer to the original tone of the post. There's also times when another word suits better as a translation. Finally, I marked the translations that are incorrect, and are changing the meaning of the sentence. Take what you like, leave the rest. ✌️

**Comment by jblancogl** [comment]
Hi @sorodrigo! I write you a short message just to thank you for the help. Today I’ll check what you've sent me and I'll merge it

Mil gracias!

**Comment by chrisbolin** [comment]
@jblancogl and @sorodrigo thank you so much!!! I'm sorry I didn't see this until now

**Comment by chrisbolin** [comment]
oh nice! @jblancogl did you merge in @sorodrigo's suggestions? that makes it so much easier for me. 👏👏 mi español es no bueno

**Comment by jblancogl** [comment]
@chrisbolin yes, i did it. all the suggestions were correct and improved the translation. thank you both again!

**Comment by chrisbolin** [comment]
It's live! https://chris.bolin.co/offline/

(make sure you dump your cache)

image

**Comment by jblancogl** [comment]
I can see it! great!

**Comment by sorodrigo** [comment]
Thanks!! aweeesome

**Pull request closed without merging**